### ISSUE TITLE
Placed chest with WorldEdit is not openable

### ISSUE THREAD

**Issue by ghost** [cuberite/WorldEdit#51]
Hello!

I have maked a schematic file with this plugin and placed it again. I am not able to open the chest and If I break it in survival mode no items appears, is this a known bug?

EDIT: After a server restart, I was able to open it - but it has no items

Reproduce:
1) Create a chest place items in it
2) Copy it with worldedit
3) Paste it
4) Try to open it

**Comment by Seadragon91** [comment]
I think this can be closed, because this has to be done in cuberite.

**Comment by NiLSPACE** [comment]
Nah. This way people know it's a known bug.

**Comment by HelenaKitty** [comment]
I just found this bug, was about to report it, thank very very much!

**Comment by HelenaKitty** [comment]
This is most likely a Cuberite issue, than a WorldEdit issue as I had a similar problem when loading an Anvil world in Cuberite.

**Comment by NiLSPACE** [comment]
You're right that it isn't a WorldEdit issue, but not being able to open chests from a vanilla world is strange since Cuberite uses the same format.

**Comment by HelenaKitty** [comment]
I thought Cuberite had three storage schemas, or have I misunderstood?

**Comment by mathiascode** [comment]
https://book.cuberite.org/#3.4

**Comment by HelenaKitty** [comment]
I vaguely remember a time when you didn't even support Anvil, is Default just Anvil?

**Comment by mathiascode** [comment]
Default is Anvil. Cuberite had its own storage schema called Compact many years ago.

**Comment by HelenaKitty** [comment]
That was it, thanks for the reminder!
